pymongo/errors.py

```diff
@@ -146,9 +146,16 @@ def __init__(self, error, code=None, details=None):
             error_labels = details.get('errorLabels')
         super(OperationFailure, self).__init__(
             error, error_labels=error_labels)
+        self.__error = error
```

Comment:
What do you think about using  `self._message` from the super class (PyMongoError) instead of adding a new attribute?